pkg/cmd/roachtest/sqlalchemy.go

```diff
@@ -157,10 +178,9 @@ func runSQLAlchemy(ctx context.Context, t test.Test, c cluster.Cluster) {
 	// Note that this is expected to return an error, since the test suite
 	// will fail. And it is safe to swallow it here.
 	rawResults, _ := c.RunWithBuffer(ctx, t.L(), node,
-		`cd /mnt/data1/sqlalchemy/ && pytest --maxfail=0 \
-		--requirements=sqlalchemy_cockroachdb.requirements:Requirements \
+		`cd /mnt/data1/sqlalchemy-cockroachdb/ && pytest --maxfail=0 \
```

Comment:
nice, so this aligns more with how the sqlalchemy-cockroachdb CI works? lgtm as long as @gordthompson can confirm

Reply:
Note that running

pytest … test/test_suite_sqlalchemy.py

will only run the SQLAlchemy tests, excluding the Alembic tests and your own specific tests (test_introspection.py, etc.). To run all of the tests just do

pytest --maxfail=0 --dburi=cockroachdb://root@localhost:26257/defaultdb?sslmode=disable

Reply:
That makes sense, I have it on my radar to do that as a followup, as it could require updating some blocklists that we maintain on our end.